### PR TITLE
filter import resources use labelSelector

### LINKED ISSUE #142: filtering resources when import cluster
Idea from https://github.com/kubernetes-sigs/kube-scheduler-simulator/issues/141
Currently when we import resources on existing clusters, we try to import all resources on that cluster.

It would be nice to be able to define filters to determine which resources to import.

/kind feature

### PR DESCRIPTION
**Pull request by YamasouA**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
when we import resources on existing clusters, we can import resources only selected labels. 

#### Which issue(s) this PR fixes:

Fixes #142 

#### Special notes for your reviewer:

/label tide/merge-method-squash

### CODE REVIEW

**Review by sanposhiho — COMMENTED** [review]
/ok-to-test

**Review by sanposhiho — COMMENTED** [review]
Also, we need a document update at https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/master/simulator/docs/import-cluster-resources.md

**Comment by LY-today** [comment]
Is there no one to advance this MR?

File: simulator/docs/import-cluster-resources.md [hunk 1]
```diff
@@ -10,10 +10,14 @@ To use this, you need to follow these two steps in the simulator configuration:
 - Set `true` to `externalImportEnabled`.
 - Set the path of the kubeconfig file for the your cluster to `KubeConfig`. 
   - This feature only requires the read permission for resources.
+- Set labels which you want to import to `importLabel`
```
Comment (sanposhiho) [inline simulator/docs/import-cluster-resources.md:13]:
```suggestion
- [optional] Set a label selector at `importLabel` if you want to import specific resources only.
```

File: simulator/config/config.go [hunk 2]
```diff
@@ -37,6 +37,8 @@ type Config struct {
 	// ExternalImportEnabled indicates whether the simulator will import resources from a target cluster once
 	// when it's started.
 	ExternalImportEnabled bool
+	// ImportLabel is Labels to select which resources import from external cluster.
+	ImportLabel map[string]string
```
Comment (sanposhiho) [inline simulator/config/config.go:41]: can we use `metav1.LabelSelector`?
https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/types.go#L1287

**Comment by YamasouA** [comment]
@sanposhiho 
I fixed to use metav1.LabelSelector.

**Review by sanposhiho — COMMENTED** [review]
just left minor points, others look great

**Comment by YamasouA** [comment]
@sanposhiho 
I rename labelSelector.

**Review by sanposhiho — APPROVED** [review]
/lgtm
/approve

**Pull request merged**